test/test_example.py

```diff
@@ -0,0 +1,12 @@
+import pytest
```

Comment:
Wenn du hier einen Demo Test hinzufügst, würde ich dich bitten, den gleichen auch demo_package hinzuzufügen. Die beiden Packages sollen immer die gleiche Funktionalität haben.

Reply:
wie meinst du das? also den gleichen test kann ich in c ja nicht hinzufügen.

Reply:
Natürlich einen Demo Test mit gleicher Funktionalität in C++ mit Google Test nachbauen.

Reply:
gut dann nehme ich den test wieder raus. Und mach da einfach ein return 1 oder so rein, weil der hat ja eh keinen sinn außer die coverage generation zu demonstrieren.

Reply:
Ganz raus wäre mir lieber, sonst haben wir nur toten Code.

Reply:
dann sag das doch bitte gleich jetzt hab ich den code in C hinzugefügt...